src/sorcha/readers/OrbitAuxReader.py

```diff
@@ -51,21 +52,46 @@ def _process_and_validate_input_table(self, input_table, **kwargs):
         # Do standard CSV file processing
         super()._process_and_validate_input_table(input_table, **kwargs)
 
+        if len(input_table) == 0:
+            return input_table
+
         pplogger = logging.getLogger(__name__)
 
-        # Check for an H column (which should not be in the orbit file).
-        if "H" in input_table.columns:
+        if not "FORMAT" in input_table.columns:
```

Comment:
Minor point - I find it a little easier to read as: `if "FORMAT" not in input_table.columns:`